appium-uiautomator2-server/build.gradle

```diff
@@ -35,5 +40,25 @@ dependencies {
     androidTestCompile 'com.android.support.test.uiautomator:uiautomator-v18:2.1.2'
     androidTestCompile 'com.android.support.test:testing-support-lib:0.1'
     androidTestCompile 'com.android.support.test.espresso:espresso-core:2.0'
+    androidTestCompile 'com.squareup.okhttp:okhttp:2.5.0'
+
+}
 
+task installAUT(type: Exec, dependsOn: ['installDebug', 'installDebugAndroidTest']) {
```

Comment:
If `assembleDebugAndroidTest` is always invoked via afterEvaluate then maybe we can drop the dependsOn. You'll have to investigate the gradle call graph to find out what it's doing. Gradle is kind of magical.